Doc/library/decimal.rst

```diff
@@ -897,6 +897,42 @@ Decimal objects
       :const:`Rounded`.  If given, applies *rounding*; otherwise, uses the
       rounding method in either the supplied *context* or the current context.
 
+   .. method:: __round__(n=None)
```

Comment:
It would be best if you could change this to use the `.. describe::` directive instead.
```suggestion
   .. describe:: __round__(n=None)
```

Reply:
`round` is not a method, it needs the Decimal as an argument, e.g.:
```suggestion
   .. describe:: round(number, ndigits=None)
```

and change *self* to *number* in the text below.

(And *ndigits* is probably better than *n* -- you can actually call `round(x, ndigits=2)`)

Reply:
Thanks for the comments. I made the requested changes.